db/controllers/controllers.js

```diff
@@ -7,6 +7,7 @@ const { fetchTopics ,
         insertComment} = require("../models/models");
 
 const {fetchArticleByIds} = require("../models/article.models")
+const {fetchTopicsBySlug} = require("../models/topics.models")
```

Comment:
This has been required in but hasn't been used?

Reply:
Thanks, I was trying to do too much, realised SQL would handle this.